apps/papra-client/src/modules/organizations/pages/organizations.page.tsx

```diff
@@ -14,6 +17,42 @@ export const OrganizationsPage: Component = () => {
     queryFn: fetchOrganizations,
   }));
 
+  // Create queries for stats of each organization
+  const statsQueries = createQueries(() => ({
+    queries: query.data?.organizations.flatMap(org => [
+      {
+        queryKey: ['organizations', org.id, 'members'],
+        queryFn: () => fetchOrganizationMembers({ organizationId: org.id }),
+        enabled: !!query.data?.organizations.length,
+      },
+      {
+        queryKey: ['organizations', org.id, 'documents', 'stats'],
+        queryFn: () => getOrganizationDocumentsStats({ organizationId: org.id }),
+        enabled: !!query.data?.organizations.length,
+      },
+    ]) ?? [],
+  }));
+
+  const getOrganizationStats = (organizationId: string) => {
+    if (!query.data?.organizations.length) return null;
+    
+    const orgIndex = query.data.organizations.findIndex(org => org.id === organizationId);
+    if (orgIndex === -1) return null;
+    
+    const membersQueryIndex = orgIndex * 2;
+    const documentsQueryIndex = orgIndex * 2 + 1;
+    
+    const membersQuery = statsQueries[membersQueryIndex];
+    const documentsQuery = statsQueries[documentsQueryIndex];
+    
+    return {
+      membersCount: (membersQuery?.data as any)?.members?.length ?? 0,
+      documentsCount: (documentsQuery?.data as any)?.organizationStats?.documentsCount ?? 0,
+      documentsSize: (documentsQuery?.data as any)?.organizationStats?.documentsSize ?? 0,
+      isLoading: membersQuery?.isLoading || documentsQuery?.isLoading,
+    };
+  };
```

Comment:
Maybe extract the organization card in a dedicated component with it's query, to have a single query to manage 

On the other hand we can add a single api endpoint `/api/organizations/statistics` to prevent having that many request and unburden the db